src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Constructors.jl

```diff
@@ -129,3 +129,7 @@ function reduced_scheme(X::AbsProjectiveScheme)
   set_attribute!(Xred, :is_reduced=>true)
   return Xred
 end
+
+function reduced_scheme(X::AbsProjectiveScheme{S,T}) where {S, T<:MPolyDecRing}
+  return X
+end
```

Comment:
I do not see, where the radical computation is hidden here? E.g. the ideal < x^2 > is a perfectly fine ideal in a standard graded polynomial ring in 2 variables, but it is not radical.

By the way: respect is_reduced (also a few lines above in the other function reduced_scheme)

Reply:
`X` is projective space by type dispatch. Nothing to compute here.